.github/workflows/build.yml

```diff
@@ -34,10 +34,6 @@ jobs:
       - uses: actions/checkout@v3
       - uses: mamba-org/setup-micromamba@v1
         with:
-          condarc: |
-            channels:
-              - conda-forge
-            channel_priority: strict
```

Comment:
Pretty sure we don't need these based on them saying:

> If nothing is specified, setup-micromamba will create a .condarc next to the micromamba binary with conda-forge as the only channel.

So it should only use conda-forge I think?

Reply:
Sounds reasonable to me!